articles/governance/policy/how-to/remediate-resources.md

```diff
@@ -15,16 +15,16 @@ understand and accomplish remediation with Azure Policy.
 
 ## How remediation security works
 
-When Azure Policy runs the template in the **deployIfNotExists** policy definition, it does so using
+When Azure Policy runs the template in a **deployIfNotExists** policy definition or runs **modify operations**, it does so using
 a [managed identity](../../../active-directory/managed-identities-azure-resources/overview.md).
-Azure Policy creates a managed identity for each assignment, but must have details about what roles
+Azure Policy allows the option to either create (system assigned) or select (user assigned) a managed identity  for each assignment, but the identity must have details about what roles
```

Comment:
```suggestion
Policy assignments can either use a system assigned managed identity that is created by the policy service or a user assigned identity provided by the user.  The managed identity needs to be granted the appropriate roles required for remediating resources
```

Reply:
Or something along that lines... "but the identity must have details about what roles to grant the managed identity" is just weird IMO.

Reply:
Do we need to mention that the identity does not need to have the appropriate roles assigned to it IF we are adding a managed identity through the portal?